Makefile

```diff
@@ -60,6 +60,11 @@ gen-swagger: ## generates the swagger documentation
 generate: ## autogenerate go files for config
 	go generate ./config/configuration.go
 
+install-subkey: ## installs subkey
+	curl https://getsubstrate.io -sSf | bash -s -- --fast
+	export PATH="$HOME/.cargo/bin:$PATH"
```

Comment:
I believe make will open a new terminal session, so export will not work. You might have to add this to the travis file.